install.sh

```diff
@@ -36,18 +36,20 @@ test -w "$HOME/$CONFIG_FILE" &&
   cp -a "$HOME/$CONFIG_FILE" "$HOME/$CONFIG_FILE.bak" &&
   echo -e "\033[0;32mYour original $CONFIG_FILE has been backed up to $CONFIG_FILE.bak\033[0m"
 
-cp "$HOME/.bash_it/template/bash_profile.template.bash" "$HOME/$CONFIG_FILE"
+cp "$BASH_IT/template/bash_profile.template.bash" "$HOME/$CONFIG_FILE"
+sed -i "s|{{BASH_IT}}|$BASH_IT|" "$HOME/$CONFIG_FILE"
+exit
```

Comment:
Haha, good catch. I had that in while testing out the sed; I'll patch it tonight